src/electron/views/report/unified-details-section.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import { CommentIcon } from 'common/icons/comment-icon';
+import { DateIcon } from 'common/icons/date-icon';
+import { UrlIcon } from 'common/icons/url-icon';
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+import { DetailsSectionProps } from 'reports/components/report-sections/details-section';
+
+export const UnifiedDetailsSection = NamedFC<DetailsSectionProps>('DetailsSection', props => {
+    const { targetAppInfo, description, scanDate, toUtcString } = props;
+
+    const createListItem = (
+        icon: JSX.Element,
+        label: string,
+        content: string | JSX.Element,
+        contentClassName?: string,
+    ) => (
+        <li>
+            <span className="icon" aria-hidden="true">
+                {icon}
+            </span>
+            <span className="screen-reader-only">{label}</span>
+            <span className={css('text', contentClassName)}>{content}</span>
+        </li>
+    );
+
+    const scanDateUTC: string = toUtcString(scanDate);
+    const showCommentRow = !!description && description !== '';
+
+    return (
+        <div className="scan-details-section">
+            <h2>Scan details</h2>
+            <ul className="details-section-list">
+                {createListItem(<UrlIcon />, 'connected device name:', targetAppInfo.device)}
```

Comment:
As far as I can tell, the only difference between this and the other details-section is this one line that mentions the targetAppInfo. That likely means this is a good opportunity to re-factor and extract that first line, re-use the component, and avoid duplication.